### PR TITLE
Feature/docs

### PR DESCRIPTION
**Pull request by azdanov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,9 +45,12 @@
     "zoe": "./bin/zoe.js"
   },
   "directories": {
-    "man": "./dist/man"
+    "man": "./dist/man",
+    "doc": "./dist/doc",
+    "html": "./dist/html"
   },
   "dependencies": {
+    "@types/opn": "^5.1.0",
```
Comment (jorgegonzalez) [inline package.json]: Move to `devDependencies`

File: utilities/docgen.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
+/* eslint-disable import/no-extraneous-dependencies, no-shadow */
```
Comment (jorgegonzalez) [inline utilities/docgen.ts]: Move these to a new folder called `scripts`, and then into a subfolder called `docs`, so `scripts/docs/docgen.ts`, same for the styles

Comment (azdanov) [reply utilities/docgen.ts]: I wonder where to put `utilities/typings/`. They belong in `scripts/typings/` probably. Right now they provide `any` stub for dependencies. Will fill them out when I have time.

Comment (jorgegonzalez) [reply utilities/docgen.ts]: Yeah move it to `scripts/typings` for now.

**Comment by jorgegonzalez** [comment]
I'll merge when CI passes.

**Pull request merged**